### ISSUE TITLE
Transform positions

### ISSUE THREAD

**Issue by Triah** [MegaCakeEater/Mnemophobia#2]
There are some wierd stuff going on with positions, nothing is really synched up with the supposed center of the plane

**Comment by MegaCakeEater** [comment]
I can't say I understand what the issue is, but if you do not intend to fix it, can you explain it and provide an example?

**Comment by Triah** [comment]
Alrighty in the attached images the positions of the sphere and the cylinder is supposed to be the exact same, yet it is not, this is especially bad with the ground plane and the other items because the floor plane is miles from the objects but is set to position 0,0,0 which should be the center of the screen, yet resetting anything does by no means center them at the central position of the plane
![cylinder position](https://user-images.githubusercontent.com/12065482/42752859-820ef36e-88ef-11e8-9c02-163ac88b89c3.PNG)
![sphere position](https://user-images.githubusercontent.com/12065482/42752868-86cefa7a-88ef-11e8-91bb-86e481d4d833.PNG)

**Comment by MegaCakeEater** [comment]
As far as i understand with my limited unity knowledge, the position of the cylinder is in relation to its parent GameObject, so try comparing those instead.

**Comment by Triah** [comment]
Yep thats how it works, thats pretty fucking hard to work with though so im gonna reset the parents so everything is aligned

**Comment by Triah** [comment]
hmm cant get it to align properly still, the stuff is just being put into the lower left corner of the plane now xD thats fairly annoying but ill look more into it once i finish the enemy i suppose

**Comment by Triah** [comment]
I think its the wierd material that is used for the plane that is the issue, its like it synchs at the corner instead of the middle, we need to find a way to make it consistent with all objects or delete the floor material and use a plane for now
